README.md

```diff
@@ -168,6 +168,14 @@ Make a new atomic batch that is either fully processed or not processed at all.
 
 If you have several inserts and deletions then a batch can be much faster.
 
+Batch is snapshotted at creation. Outside writes are not taken into account when reading.
```

Comment:
Not highly true:
```js
const batch = await db.batch()
t.is(batch.version, 1)
await db.put('/a')
t.is(batch.version, 2)
console.log(await batch.get('/a')) // => { seq, key, value }
```

So state is not snapshotted?

Reply:
A put (obvs) updates the snap.

Reply:
It's a `db.put` external write outside of batch, is that intended to update the snap of the batch?

Reply:
Ah db my bad. I'd prefer to leave this be for now. Its due to update being called. This needs some (breaking) refactor to remove all update calls